feature: Add tag prefix documentation (#27)
codacy/pulse-user-docs: docs/one-click-integrations.md

!!! important
    Consider the following before using the GitHub integration:

>     -   The integration obtains information about deployments from Git tags that follow the [SemVer](https://semver.org) convention, excluding pre-release versions, but allowing release prefixes. Valid tags e.g.: `1.0.0`, `v2.3.4`

Reviewer: Small suggestion to simplify the punctuation:

```suggestion
    -   The integration obtains information about deployments from Git tags that follow the [SemVer](https://semver.org) convention, excluding pre-release versions but allowing release prefixes. For example, `1.0.0` or `v2.3.4`.
```

I really like the idea of providing examples! :+1:
